tests/Spec.hs

```diff
@@ -2818,6 +2888,20 @@ main = hspec $ do
       mergeWindowExemption `shouldBe` ["hoffbot"]
       Config.commentPrefix trigger `shouldBe` "@hoffbot"
 
+    it "correctly assumes a missing 'safeForFriday' field to be Nothing" $ do
+      -- This test makes sure the new project configuration field, `safeForFriday`
+      -- is correctly parsed into Nothing when it's absent from the JSON file.
+      Right cfg <- Config.loadConfiguration "package/example-config.json"
+      let project = head $ Config.projects cfg
```

Comment:
Maybe nice to check that it's the right project by i.e. checking the name of the project.